Utilizr.Vpn.Ras/RasNativeProvidor.cs

```diff
@@ -78,6 +80,39 @@ public string? CurrentServer
         private ManualResetEvent _connectDone = new ManualResetEvent(false);
         private Task _checkStatusTask;
 
+        private const uint SC_MANAGER_CONNECT = 0x0001;
+        private const uint SERVICE_QUERY_CONFIG = 0x00000001;
+        private const uint SERVICE_CHANGE_CONFIG = 0x00000002;
+        private const uint SERVICE_NO_CHANGE = 0xFFFFFFFF;
+
+        private enum ServiceStartupType : uint
+        {
+            /// <summary>
+            /// A device driver started by the system loader. This value is valid only for driver services.
+            /// </summary>
+            BootStart = 0,
+
+            /// <summary>
+            /// A device driver started by the IoInitSystem function. This value is valid only for driver services.
+            /// </summary>
+            SystemStart = 1,
+
+            /// <summary>
+            /// A service started automatically by the service control manager during system startup.
+            /// </summary>
+            Automatic = 2,
+
+            /// <summary>
+            /// A service started by the service control manager when a process calls the StartService function.
+            /// </summary>
+            Manual = 3,
+
+            /// <summary>
+            /// A service that cannot be started. Attempts to start the service result in the error code ERROR_SERVICE_DISABLED.
+            /// </summary>
+            Disabled = 4
+        }
+

```

Comment:
This can be moved to `Utilizr\Win32\Advapi32\Flags` to try and keep it organised.